src/dash-table/components/Export/index.tsx

```diff
@@ -27,12 +42,12 @@ export default React.memo((props: IExportButtonProps) => {
 
         const maxLength = getHeaderRows(columns);
         const heading = (export_headers !== ExportHeaders.None) ? createHeadings(columnHeaders, maxLength) : [];
-        const ws = createWorksheet(heading, virtual_data.data, columnID, export_headers, merge_duplicate_headers);
+        const ws = createWorksheet(heading, filterData(virtual_data.data, columnID), columnID, export_headers, merge_duplicate_headers);
         const wb = createWorkbook(ws);
         if (export_format === ExportFormat.Xlsx) {
-            XLSX.writeFile(wb, 'Data.xlsx', {bookType: 'xlsx', type: 'buffer'});
+            XLSX.writeFile(wb, 'Data.xlsx', { bookType: 'xlsx', type: 'buffer' });
```

Comment:
These diffs come from the linter -- I can remove them if necessary.

Reply:
I think this PR is redundant with what's been done here https://github.com/plotly/dash-table/pull/592. See https://github.com/plotly/dash-table/blame/dev/src/dash-table/components/Export/utils.tsx#L59 for column filtering.